src/js/components/agency/statusOfFunds/VisualizationSection.jsx

```diff
@@ -52,22 +57,132 @@ const VisualizationSection = ({
+                    {data.name}
+                </div>
+            ),
+            (
+                <div>
+                    {formatMoneyWithPrecision(data._outlays)}
+                </div>
+            )
+        ]
+        :
+        [
+            (
+                <div>
+                    {data.name}
+                </div>
+            ),
+            (
+                <div>
+                    {formatMoneyWithPrecision(data._budgetaryResources)}
+                </div>
+            ),
+            (
+                <div>
+                    {formatMoneyWithPrecision(data._obligations)}
+                </div>
+            )
+        ]));
+
+    const changeView = (label) => {
+        setViewType(label);
+    };
+
+    const chartTableToggle = (
+        <ChartTableToggle
+            activeType={viewType}
+            changeView={changeView} />
+    );
+
     return (
         <div
             className="status-of-funds__visualization"
             onMouseLeave={() => {
                 const el = document.querySelector("div.tooltip-wrapper.sof_chart-tt");
                 el.style.display = "none";
             }}>
-            <h6>{level === 1 ? selectedSubcomponent?.name : agencyName} by <span className="status-of-funds__emphasis">{levels[level]}</span> for FY {fy}</h6>
-            <div
-                className="status-of-funds__controls">
-                <RoundedToggle toggle={toggle} onKeyToggle={onKeyToggle} onToggle={onToggle} label="View Outlays" />
-                <div className="status-of-funds__line-div" />
-                <div className="status-of-funds__accordion">
-                    <Accordion setOpen={setOpen} closedIcon="chevron-down" openIcon="chevron-up" title="What is this?" />
-                </div>
-            </div>
+            {isMobile ? (
+                <>
+                    <h6>{level === 1 ? selectedSubcomponent?.name : agencyName} by <span className="status-of-funds__emphasis">{levels[level]}</span> for FY {fy}
+                    </h6>
+                    <div className="status-of-funds__controls-mobile">
+                        <div className="status-of-funds__controls-mobile-row-one">
+                            <RoundedToggle toggle={toggle} onKeyToggle={onKeyToggle} onToggle={onToggle} label="View Outlays" />
+                            {chartTableToggle}
+                        </div>
+                        <Accordion setOpen={setOpen} closedIcon="chevron-down" openIcon="chevron-up" title={accordionTitle} />
+                    </div>
+                </>
+            )
+                :
+                (
+                    <>
+                        <div className="status-of-funds__controls">
+                            <div className="status-of-funds__controls-desktop-row-one">
+                                <h6>{level === 1 ? selectedSubcomponent?.name : agencyName} by <span className="status-of-funds__emphasis">{levels[level]}</span> for FY {fy}
```

Comment:
see above

Reply:
we decided it's better to leave these changes for another ticket which will address more of these type of issues all at once